.gitattributes

```diff
@@ -28,11 +28,7 @@
 /.gitignore         export-ignore
 /.scrutinizer.yml   export-ignore
 /.travis.yml        export-ignore
```

Comment:
There's no travis config. Can remove it.

Reply:
done.